### PR TITLE
[markdown-it]: specify dependency versions on markdown-it v13

### PR DESCRIPTION
**Pull request by brc-dd**
Please fill in this template.

refer #69355

- [ ] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [ ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:

- [ ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ ] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [ ] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:

- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the `package.json`.

If removing a declaration:

- [ ] If a package was never on Definitely Typed, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

File: types/markdown-it/package.json [hunk 1]
```diff
@@ -16,8 +16,8 @@
         }
     },
     "dependencies": {
-        "@types/linkify-it": "*",
-        "@types/mdurl": "*"
+        "@types/linkify-it": "^5",
+        "@types/mdurl": "^2"
```
Comment (jakebailey) [inline types/markdown-it/package.json:20]: These look correct to me given upstream's dependencies.

File: types/markdown-it/v13/package.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
+{
+    "private": true,
+    "name": "@types/markdown-it",
+    "version": "13.0.9999",
+    "projects": [
+        "https://github.com/markdown-it/markdown-it"
+    ],
+    "dependencies": {
+        "@types/linkify-it": "^3",
```
Comment (jakebailey) [inline types/markdown-it/v13/package.json:9]: This should be ^4, given https://unpkg.com/markdown-it@13.0.2/package.json says linkify-it at 4.

Comment (brc-dd) [reply types/markdown-it/v13/package.json:9]: There are no types corresponding to v4 - https://www.npmjs.com/package/@types/linkify-it?activeTab=versions

Comment (jakebailey) [reply types/markdown-it/v13/package.json:9]: Bleh, okay. I guess the other PR jumped right to 5.

**Review by jakebailey — APPROVED** [review]

File: types/markdown-it/v13/package.json (hunk 2 above)
Comment (brc-dd) [reply types/markdown-it/v13/package.json:9]: v4 didn't had any type changes though - https://github.com/markdown-it/linkify-it/blob/master/CHANGELOG.md

**Comment by brc-dd** [comment]
Ready to merge

**Pull request merged**